MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift

```diff
@@ -40,16 +49,24 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
         return 1
     }
 
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 190
+    }
+
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: "reuseTableCell", for: indexPath) as? NewTableViewCell else {
-            fatalError("Unable to dequeue NewTableViewCell.")
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Identifiers.reuseTableCell, for: indexPath) as? NewTableViewCell else {
+            return fetchDefaultTableCell()
         }
         cell.configure(with: nowPlayingViewModel.nowPlayingMovies)
-
+        cell.didSelectItem = { [weak self] movie in
```

Comment:
There is a table view function called did select cell t/ow at. This should be done in there.😄

Reply:
I'm not entirely sure about how to do that, because in here it is the cell of the collection view that must be clicked, not the tableView cell, because the table view cell holds the collection view as a whole, so this way the user can click on the collection view cell individually instead of the tableview cell holding the collection view